parm/config/config.defaults.s2sw

```diff
@@ -16,13 +16,13 @@ use_cice_alb=".true."
 
 # config.wave
 
-waveGRD='mx025'
```

Comment:
Were ICs added so that this will run the C384 tests out of the box?  We need to add ICs or add this as an option instead of the default.  We really need to add a more configurable wave grid option as GFS needs this to be mx025 versus the GEFS option.

Reply:
If this is just for GEFS, the setting changes in this file should be made to GEFS and not here. Other changes to handle the grid should be here though.

CC: @XianwuXue-NOAA

Reply:
Change the settings in _this_ file back. You can work with @XianwuXue-NOAA to get them as the default in GEFS. To run g-w directly with the GEFS grid, people will have to manually edit the config file.

Reply:
FYI -   I am going to move several of these variables to config.ufs for each wave grid, that way when someone switches between grids, we have this set of grid dependent variables set and only waveGRD then needs to be set.   Fix file update which is required before this and the other changes can go in will be requested shortly, with a PR to @NeilBarton-NOAA's branch for this PR to update things for the gefs grid.

Reply:
@JessicaMeixner-NOAA As long as they are variables ONLY used by the forecast job, since `config.ufs` isn't used by anything else, while `config.wave` (which these are overriding) is used in several jobs. And, if you are moving anything to `config.ufs` from this section, it should be removed from `config.wave` as well (or wherever the setting noramlly is).

With our move towards running S2S more typically, we should also probably look to retire this config and fold the settings back in to other places (in a future PR).

Reply:
Glad I mentioned this as these variables are used elsewhere and this will save some time as those variables are used for things other than the forecast.   I am going to have some variable groups defined based on waveGRD, which will help in switching grids.   TBD if it's this file or config.wave or where.  Thank you @WalterKolczynski-NOAA !!!